arbnode/forwarder.go

```diff
@@ -11,36 +11,51 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/offchainlabs/nitro/util/redisutil"
+	"github.com/offchainlabs/nitro/util/stopwaiter"
 	"github.com/pkg/errors"
 	flag "github.com/spf13/pflag"
 
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/log"
 	"github.com/ethereum/go-ethereum/rpc"
 )
 
 type ForwarderConfig struct {
 	ConnectionTimeout     time.Duration `koanf:"connection-timeout"`
 	IdleConnectionTimeout time.Duration `koanf:"idle-connection-timeout"`
 	MaxIdleConnections    int           `koanf:"max-idle-connections"`
+	RedisUrl              string        `koanf:"redis-url"`
+	UpdateInterval        time.Duration `koanf:"update-interval"`
+	RetryInterval         time.Duration `koanf:"retry-interval"`
 }
 
 var DefaultTestForwarderConfig = ForwarderConfig{
 	ConnectionTimeout:     2 * time.Second,
 	IdleConnectionTimeout: 2 * time.Second,
 	MaxIdleConnections:    1,
+	RedisUrl:              redisutil.DefaultTestRedisURL,
+	UpdateInterval:        time.Millisecond * 10,
+	RetryInterval:         time.Millisecond * 3,
 }
 
 var DefaultNodeForwarderConfig = ForwarderConfig{
 	ConnectionTimeout:     30 * time.Second,
 	IdleConnectionTimeout: 15 * time.Second,
 	MaxIdleConnections:    1,
+	RedisUrl:              "",
+	UpdateInterval:        time.Duration(5) * time.Second,
+	RetryInterval:         time.Second,
```

Comment:
Let's make `UpdateInterval` 1 second and `RetryInterval` 100ms. Redis calls are cheap and we don't want to send a tx to a sequencer after it shuts down.